formio/formio
# Add minimum docker support

Hi, it took me a couple hours to figure this out, so I figured I would share. maintainers feel free to request further modifications/explanation. 
Cheers!

## Description 

My goal was to not have to install MongoDB/Node on my system, and just use Docker. 

## Steps to test

1. Install docker and docker-compose
1. `cd formio`
1. `docker-compose up -d mongo`
1. `docker-compose run formio` this should ask you the install/startup questions in your terminal
1. [`http://localhost:3001/`](http://localhost:3001/)

## Impacts

* Includes Dockerfile for getting all dependencies and running formio
* Includes docker-compose.yml to start MongoDB and formio on the host
* Added one line to README to mention docker support
